### ISSUE TITLE
obsolete example in Readme?

### ISSUE THREAD

**Issue by WBurzynski** [tinyobjloader/tinyobjloader#288]
**Describe the issue**
I got tinyobjloader 1.0.6 from conan repository without any options specified and tried to run `tinyobj::LoadObj`, copied from README, however i got error, "no matching function for call...". It seems API changed. Is there a documentation for this library, or just examples? Or maybe this is issue with package?

**To Reproduce**
1. get tinyobjloader/1.0.6 via conan
2. paste code from README
3. try to compile code

**Comment by syoyo** [comment]
Conan package is not recommended to use since tinyobjloader is a header only library.

Recent conan support is provided by a contributor and v1.0.6 is too old. Please use `master`(`main`) branch of tinyobjloader repo directly.

**Comment by WBurzynski** [comment]
Ok, thank you for the answer. But still i'd like ask if there is documentation available, or just examples and scrolling through  header?

**Comment by Namburger** [comment]
@WBurzynski I ran into this problem too, basically the 1.0.6 release did not have the warn parameter.
The signature of the LoadObj looks like this:
```
bool LoadObj(attrib_t *attrib, std::vector<shape_t> *shapes,
             std::vector<material_t> *materials, std::string *err,
             const char *filename, const char *mtl_basedir = NULL,
             bool triangulate = true);
```
I then found out that warning and error are both puts in the err parameter. 

This is changed in later version and so the example document is not obsolete at all, it's the 1.0.6 that is not up to date with the doc.
